src/aks-preview/azext_aks_preview/decorator.py

```diff
@@ -135,6 +140,29 @@ def init_nat_gateway_models(self) -> None:
         # for model_name, model_type in nat_gateway_models.items():
         #     setattr(self, model_name, model_type)
 
+    def init_pod_identity_models(self) -> None:
+        """Initialize models used by nat gateway.
+
+        The models are stored in a dictionary, the key is the model name and the value is the model type.
+
+        :return: None
+        """
+        pod_identity_models = {}
+        pod_identity_models["ManagedClusterPodIdentityProfile"] = self.__cmd.get_models(
+            "ManagedClusterPodIdentityProfile",
+            resource_type=self.resource_type,
+            operation_group="managed_clusters",
```

Comment:
nit: can we make this `managed_clusters` operation group as a const?

Reply:
Will try the @property decorator.

Reply:
Changed in commit [#4243 9e1827e](https://github.com/Azure/azure-cli-extensions/pull/4243/commits/9e1827e8f5150a8f3e3884c433c9d9e35f668c3d).